platform-server-kernel/src/main/java/uno/cod/platform/server/core/service/OrganizationMemberService.java

```diff
@@ -0,0 +1,64 @@
+package uno.cod.platform.server.core.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uno.cod.platform.server.core.domain.Organization;
+import uno.cod.platform.server.core.domain.OrganizationMember;
+import uno.cod.platform.server.core.domain.OrganizationMemberKey;
+import uno.cod.platform.server.core.domain.User;
+import uno.cod.platform.server.core.dto.organizationMember.OrganizationMemberCreateDto;
+import uno.cod.platform.server.core.repository.OrganizationMemberRepository;
+import uno.cod.platform.server.core.repository.OrganizationRepository;
+import uno.cod.platform.server.core.repository.UserRepository;
+
+import javax.transaction.Transactional;
+
+@Service
+@Transactional
+public class OrganizationMemberService {
+    private final OrganizationMemberRepository repository;
+    private final UserRepository userRepository;
+    private final OrganizationRepository organizationRepository;
+
+    @Autowired
+    public OrganizationMemberService(OrganizationMemberRepository repository, UserRepository userRepository, OrganizationRepository organizationRepository) {
+        this.repository = repository;
+        this.userRepository = userRepository;
+        this.organizationRepository = organizationRepository;
+    }
+
+    public void save(OrganizationMemberCreateDto dto) {
+        User user = userRepository.findOne(dto.getUserId());
+        if (user == null) {
+            throw new IllegalArgumentException("user is not valid");
+        }
+        Organization organization = organizationRepository.findOne(dto.getOrganizationId());
+        if (organization == null) {
+            throw new IllegalArgumentException("organization is not valid");
+        }
+        OrganizationMemberKey key = new OrganizationMemberKey();
+        key.setUser(user);
+        key.setOrganization(organization);
+        OrganizationMember member = new OrganizationMember();
+        member.setKey(key);
+        member.setAdmin(dto.isAdmin());
+        member = repository.save(member);
+        user.addOrganizationMember(member);
+        organization.addOrganizationMember(member);
+    }
+
+    public void delete(OrganizationMemberCreateDto dto) {
```

Comment:
if the last one wants to leave he has to delete the organization, if the last admin wants to leave, he has to promote someone upfront (so that an organization has always an admin)